swag/db.py

```diff
@@ -211,3 +331,15 @@ def __init__(self, account):
         self.timezone = account.timezone
         self.timezone_lock_date = account.timezone_lock_date
         self.creation_date = account.creation_date
+
+
+class CagnotteInfo:
+    def __init__(self, cagnotte) -> None:
+
+        self.id = cagnotte.id
+        self.name = cagnotte.name
+        self.balance = cagnotte.balance
+        self.currency = cagnotte.currency
+        self.manager = cagnotte.manager
```

Comment:
managers

Reply:
Resolved in : https://github.com/BobbySoftwares/Bybbot/pull/7/commits/c9cfecdb424780b7243e2b2ba8fcc0fba8a01e8f